modules/extra/images.yml

```diff
@@ -244,6 +244,15 @@ images:
       - "v0.3.0"
       - "v0.4.0"
       - "v0.5.0"
+      
+    destinations:
+      - registry.sighup.io/fury/jimmidyson/configmap-reload
+
+  - name: Configmap reloader
+    source: docker.io/jimmidyson/configmap-reload
+    tag:
```

Comment:
```suggestion
```

No need to specify the same source and destination twice 😄

Reply:
It was a typo that duplicated the image object; it is the same destination but from a different source. I’ve updated the source.